khmer/khmer_args.py

```diff
@@ -389,18 +393,22 @@ def add_loadgraph_args(parser):
                         help='load a precomputed k-mer graph from disk')
 
 
-def calculate_graphsize(args, graphtype, multiplier=1.0):
+def calculate_graphsize(args, graphtype, multiplier=1.0, _sysmem=None):
```

Comment:
Attracts attention in the sense that I'd have to think about it before I could show you another example of it. `_` in python does indeed mean "don't mess" though usually apply to member variables or functions/methods.

TL;DR: There should be a better way of doing this (but I can't think of one right now short of introducing `mock` as a dependency for `python2`). Maybe an executive decision is needed re adding dependencies for developers.